EnvelopeItem/TransportConfiguration.php

```diff
@@ -99,24 +99,34 @@ public function setTimeToLive(int $timeToLive = null): self
         return $this;
     }
 
-    public function serialize()
+    public function serialize(): string
```

Comment:
Do we still need those methods?

Reply:
I left this method for supporting the previous version of the package if someone has been using this method in his code already

Reply:
deleted